packages/nav-frontend-alertstriper/package.json

```diff
@@ -12,6 +12,10 @@
     "type": "git",
     "url": "git+https://github.com/navikt/nav-frontend-moduler.git"
   },
+  "scripts": {
+    "build": "tsc -p tsconfig.json",
```

Comment:
Om tsconfig blir lik for alle pakkene, gir det da mening å peke på `../tsconfig.json` her, eller vil ikke det funke?

Reply:
Ja er litt tungvindt å måtte ha en egen tsconfig i hver pakke.
```
  "compilerOptions": {
    "outDir": "./lib",
    "rootDir": "./src"
  },
  "include": ["./src"]
```
Må være lokalisert i hver pakke for å få riktig pathinbg. Men vil gjerne ha en løsning på det så vi slipper dette!